lib/github/ldap.rb

```diff
@@ -315,6 +347,9 @@ def configure_member_search_strategy(strategy = nil)
     def active_directory_capability?
       capabilities[:supportedcapabilities].include?(ACTIVE_DIRECTORY_V51_OID)
     end
-    private :active_directory_capability?
```

Comment:
Made this public since `Domain` now [needs to pivot](https://github.com/github/github-ldap/pull/94/files#diff-e1bd3bdd1e3ab28a95efb8ba8db6cd0cR122) on whether the server is an AD or not.